lib/env-helper.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path');
+const fs = require('fs');
+const cwd = process.cwd();
+
+const parseEnvFile = (filename) => {
+  const filePath = path.resolve(cwd, filename);
+  if (!fs.existsSync(filePath)) {
+    console.warn('No env file found!');
+    return {};
+  }
+  const data = fs.readFileSync(filePath);
+  const lines = data.toString().split('\n');
+  const obj = {};
+  for (const line of lines) {
```

Comment:
does that work out? I thought `line` should be declared by `let` and not `const`